x/twap/store_test.go

```diff
@@ -82,14 +81,14 @@ func (s *TestSuite) TestGetAllMostRecentRecordsForPool() {
 		},
 		"set multi-asset pool record": {
 			recordsToSet: []types.TwapRecord{
-				newEmptyPriceRecord(1, baseTime, denomB, denomA),
-				newEmptyPriceRecord(1, baseTime, denomC, denomB),
-				newEmptyPriceRecord(1, baseTime, denomC, denomA)},
+				newEmptyPriceRecord(1, baseTime, denom1, denom0),
+				newEmptyPriceRecord(1, baseTime, denom2, denom1),
+				newEmptyPriceRecord(1, baseTime, denom2, denom0)},
 			poolId: 1,
 			expectedRecords: []types.TwapRecord{
-				newEmptyPriceRecord(1, baseTime, denomB, denomA),
-				newEmptyPriceRecord(1, baseTime, denomC, denomA),
-				newEmptyPriceRecord(1, baseTime, denomC, denomB)},
+				newEmptyPriceRecord(1, baseTime, denom1, denom0),
+				newEmptyPriceRecord(1, baseTime, denom2, denom0),
+				newEmptyPriceRecord(1, baseTime, denom2, denom1)},
 		},
```

Comment:
Might be overkill, but should we also do two multi-asset pool records, then set them in reverse order to unsure the last record, independent of time, still takes precedence?